src/terrama2/core/utility/Utils.hpp

```diff
@@ -166,6 +169,16 @@ namespace terrama2
      */
     std::unique_ptr<te::rst::Raster> multiplyRaster(const te::rst::Raster& raster, const double& multiplier);
 
+
+    size_t propertyPosition(const te::da::DataSet* dataSet, const std::string& propertyName);
+
+    /*!
+     * \brief Returns a valid name for te::da::DataSet properties name
+     * \param text The base name to process
+     * \return A valid name for te::da::DataSet properties name
+     */
+    std::string validPropertyName(std::string text);
```

Comment:
Improve the function name,
the name should show that you are creating a new name based on the older (invalid) one